<launch>	
	<node name="sensor_0_tf" pkg="rospyd7a" type="sensor_tf_publisher.py">
      <param name="receiving_frame" value="base_link"/>
      <param name="id_1" type="string" value="3191882256028378"/>
      <param name="id_1_x" type="double" value="5.0"/>
      <param name="id_1_y" type="double" value="0.0"/>
      <param name="id_1_z" type="double" value="0.0"/>
    </node>
</launch>